Tests/Helpers/TestHelpers.swift

```diff
@@ -132,7 +132,7 @@ struct TestHelpers {
             }
 
             return PingResponse(
-                nextPingMilliseconds: 0,
+                nextPingMilliseconds: Int.max,
```

Comment:
Does it mean that the next ping request will be sent after `Int.max` milliseconds?

Reply:
Yes. Putting 0 was a very big mistake which caused weird race conditions that sometimes made PublicApiSpec test fail.
(because ping calls were spawned in thousands)